.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -241,6 +241,8 @@ if [ "${WORKER}" == 0 ]; then
     chmod  770                                        ${SUBMITTY_DATA_DIR}/vcs
     chown  root:${DAEMONCGI_GROUP}                    ${SUBMITTY_DATA_DIR}/vcs/git
     chmod  770                                        ${SUBMITTY_DATA_DIR}/vcs/git
+    #add cgi user to docker group
+    usermod -a -G "docker" ${CGI_USER}
```

Comment:
This should probably be in the `install_submitty.sh` script where we do the other various group adds and such, and not here? Not sure we do any other group changes in this particular script.

Reply:
ok sounds good, I'm not too familiar with the setup scripts.